src/flash_saml21.c

```diff
@@ -0,0 +1,115 @@
+#include "uf2.h"
```

Comment:
is this exactly the same as D21?

Reply:
if so, please use a single file

Reply:
Yes, is this exactly the same as D21, but I don't know how to make the conditional

Reply:
ok, let's leave it as is